features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -172,31 +175,37 @@ Feature: SDN compoment upgrade testing
       | name=test-pods |
     And evaluation of `pod(1).ip_url` is stored in the :pod2ip clipboard
     And evaluation of `pod(0).name` is stored in the :pod1 clipboard
-    When I execute on the "<%= cb.pod1 %>" pod:
-      | curl | -s | --connect-timeout | 5 | <%= cb.pod2ip %>:8080 |
-    Then the step should fail
     Given a pod becomes ready with labels:
       | name=hello-idle |
     And evaluation of `pod(2).name` is stored in the :pod3 clipboard
     And evaluation of `pod(2).ip_url` is stored in the :pod3ip clipboard
+    And I wait up to 10 seconds for the steps to pass:
+    """
+    When I execute on the "<%= cb.pod1 %>" pod:
```

Comment:
@asood-rh did you tested with cuke5.x?  If so, do you  mind testing it again by replacing `<%= cb.pod1 %>` with `#{cb.pod1}`?

Reply:
@pruan-rht The script is working with cucumber 5.x.
gem list shows me it is 5.3.0
cucumber (5.3.0, 2.4.0)
cucumber-core (8.0.1, 1.5.0)
cucumber-create-meta (2.0.4)
cucumber-cucumber-expressions (10.3.0)
cucumber-gherkin (15.0.2)
cucumber-html-formatter (9.0.0)
cucumber-messages (13.2.1)
cucumber-tag-expressions (2.0.4)

What is the purpose of replacing <%= cb.pod1 %> with #{cb.pod1}? It did not work.